<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostListDialogClass</class>
 <widget class="QDialog" name="HostListDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ホスト一覧</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="HostTreeView" name="treeView_HostList">
       <property name="editTriggers">
        <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
       </property>
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_Connect">
         <property name="text">
          <string>接続(&amp;S)</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Close">
         <property name="text">
          <string>閉じる(&amp;O)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_1">
     <item>
      <widget class="QPushButton" name="pushButton_NewHost">
       <property name="text">
        <string>新規ホスト(&amp;N)...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_NewGroup">
       <property name="text">
        <string>新規グループ(&amp;G)...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Mod">
       <property name="text">
        <string>設定変更(&amp;M)...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Copy">
       <property name="text">
        <string>コピー(&amp;C)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Del">
       <property name="text">
        <string>削除(&amp;D)...</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_1">
       <item>
        <widget class="QPushButton" name="pushButton_Up">
         <property name="text">
          <string>↑</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Down">
         <property name="text">
          <string>↓</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Default">
       <property name="text">
        <string>既定の設定(&amp;F)...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Help">
       <property name="text">
        <string>ヘルプ</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_1">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HostTreeView</class>
   <extends>QTreeView</extends>
   <header>ui/host/hosttreeview.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>treeView_HostList</tabstop>
  <tabstop>pushButton_Connect</tabstop>
  <tabstop>pushButton_Close</tabstop>
  <tabstop>pushButton_NewHost</tabstop>
  <tabstop>pushButton_NewGroup</tabstop>
  <tabstop>pushButton_Mod</tabstop>
  <tabstop>pushButton_Copy</tabstop>
  <tabstop>pushButton_Del</tabstop>
  <tabstop>pushButton_Up</tabstop>
  <tabstop>pushButton_Down</tabstop>
  <tabstop>pushButton_Default</tabstop>
  <tabstop>pushButton_Help</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Close</sender>
   <signal>clicked()</signal>
   <receiver>HostListDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_NewHost</sender>
   <signal>clicked()</signal>
   <receiver>HostListDialogClass</receiver>
   <slot>onClick_pushButton_NewHost()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_NewGroup</sender>
   <signal>clicked()</signal>
   <receiver>HostListDialogClass</receiver>
   <slot>onClick_pushButton_NewGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Mod</sender>
   <signal>clicked()</signal>
   <receiver>HostListDialogClass</receiver>
   <slot>onClick_pushButton_Mod()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Copy</sender>
   <signal>clicked()</signal>
   <receiver>HostListDialogClass</receiver>
   <slot>onClick_pushButton_Copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Del</sender>
   <signal>clicked()</signal>
   <receiver>HostListDialogClass</receiver>
   <slot>onClick_pushButton_Del()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Up</sender>
   <signal>clicked()</signal>
   <receiver>HostListDialogClass</receiver>
   <slot>onClick_pushButton_Up()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Down</sender>
   <signal>clicked()</signal>
   <receiver>HostListDialogClass</receiver>
   <slot>onClick_pushButton_Down()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Default</sender>
   <signal>clicked()</signal>
   <receiver>HostListDialogClass</receiver>
   <slot>onClick_pushButton_Default()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Help</sender>
   <signal>clicked()</signal>
   <receiver>HostListDialogClass</receiver>
   <slot>onClick_pushButton_Help()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Connect</sender>
   <signal>clicked()</signal>
   <receiver>HostListDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectedHost(QModelIndex)</slot>
  <slot>onClick_pushButton_NewHost()</slot>
  <slot>onClick_pushButton_NewGroup()</slot>
  <slot>onClick_pushButton_Mod()</slot>
  <slot>onClick_pushButton_Copy()</slot>
  <slot>onClick_pushButton_Del()</slot>
  <slot>onClick_pushButton_Up()</slot>
  <slot>onClick_pushButton_Down()</slot>
  <slot>onClick_pushButton_Default()</slot>
  <slot>onClick_pushButton_Help()</slot>
  <slot>onClick_pushButton_Connect()</slot>
 </slots>
</ui>
